src/Constants.h

```diff
@@ -120,3 +120,15 @@ constexpr uint32_t RPLIDAR_S1_BAUDRATE = 256000;
 
 constexpr double CONTROL_HZ = 10.0;
 } // namespace Constants
+
+using namespace std::chrono_literals;
```

Comment:
We should avoid `using` statements in header files. So the `1s` below should be `std::chrono::seconds(1)`.